src/EnergyPlus/UnitarySystem.cc

```diff
@@ -6281,6 +6286,24 @@ namespace UnitarySystems {
                     }
                 } else if (UtilityRoutines::SameString(loc_m_NoCoolHeatSAFMethod, "None") || loc_m_NoCoolHeatSAFMethod == "") {
                     thisSys.m_NoCoolHeatSAFMethod = state.dataUnitarySystems->None;
+                    if (thisSys.m_ControlType == ControlType::CCMASHRAE) {
+                        if (loc_m_NoCoolHeatSAFMethod_SAFlow == -99999.0) { // no load air flow is autosized
+                            thisSys.m_MaxNoCoolHeatAirVolFlow = DataSizing::AutoSize;
+                            thisSys.m_RequestAutoSize = true;
+                        } else if (loc_m_NoCoolHeatSAFMethod_SAFlow == -999.0) { // no load air flow is blank
```

Comment:
The -99999 and -999 isn't necessarily great, but I realize it's likely just a part of the existing pattern.  It can stay.